kyo-bench/src/main/scala/kyo/bench/Bench.scala

```diff
@@ -1,17 +1,11 @@
 package kyo.bench
 
-import cats.effect.IO
-import cats.effect.unsafe.implicits.global
-import kyo.*
 import org.openjdk.jmh.annotations.*
-import zio.UIO
-import zio.ZLayer
 
 @State(Scope.Benchmark)
 @Fork(
     value = 1,
     jvmArgs = Array(
-        "-Dcats.effect.tracing.mode=DISABLED",
```

Comment:
Tracing was disabled for cats-effect to make the results more fair since Kyo didn't have tracing. The default `CACHED` config also isn't very fair because the equivalent of Kyo's tracing would be `FULL` but cats-effect has a prohibitive performance cost with it.